utils/crypto.ts

```diff
@@ -164,3 +164,29 @@ export const electrumVaultMnemonicToSeed = (mnemonic: string, password = '') =>
     keylen: 64,
     digest: 'sha512',
   });
+
+export const getMasterPublicKeyPrefix = (header: string) => {
```

Comment:
instead of type string would be nice to create the type, so TS would guard against making the typo 

`type MasterPublicKey = 'xpub' | 'ypub'...` etc.

Reply:
true